app/main.py

```diff
@@ -1,3 +1,57 @@
+from __future__ import annotations
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (int, float)) -> None:
+        if km >= 0:
+            self.km = km
```

Comment:
Bug: Your constructor doesn't handle the case where the 'km' argument is less than 0. According to your logic, if a negative value is passed, the 'km' attribute won't be set, which will lead to an AttributeError when you try to access it later. You should either raise an exception if a negative value is passed or handle it in some other way.